<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Get a specific gene metadata — getGeneMetadata • civicr</title>

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha256-916EbMg70RQy9LHiGkXzG8hSg9EdNy97GazNG/aiY1w=" crossorigin="anonymous" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script>

<!-- sticky kit -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>



<meta property="og:title" content="Get a specific gene metadata — getGeneMetadata" />

<meta property="og:description" content="Retrieve metadata for a specific gene from the CIViC DB" />
<meta name="twitter:card" content="summary" />



<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->


  </head>

  <body>
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">civicr</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
      
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      
      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Get a specific gene metadata</h1>
    
    <div class="hidden name"><code>getGeneMetadata.Rd</code></div>
    </div>

    <div class="ref-description">
    
    <p>Retrieve metadata for a specific gene from the CIViC DB</p>
    
    </div>

    <pre class="usage"><span class='fu'>getGeneMetadata</span>(<span class='no'>id</span>, <span class='no'>type</span>)</pre>
    
    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>id</th>
      <td><p>ID of the gene of interest</p></td>
    </tr>
    <tr>
      <th>type</th>
      <td><p>Type of metadata (comments, suggested_changes, revisions, variants)</p></td>
    </tr>
    <tr>
      <th>identifier_type</th>
      <td><p>Type of gene identifier (entrez_id, entrez_symbol, civic_id)</p></td>
    </tr>
    </table>
    
    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>An S3 Object of type civic_api containing the content, url, and response</p>
    

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='fu'>getGeneMetadata</span>(<span class='kw'>id</span> <span class='kw'>=</span> <span class='fl'>1</span>, <span class='kw'>type</span> <span class='kw'>=</span> <span class='st'>"comments"</span>)</div><div class='output co'>#&gt; $content
#&gt; list()
#&gt; 
#&gt; $url
#&gt; [1] "https://civicdb.org/api/genes/1/comments"
#&gt; 
#&gt; $response
#&gt; Response [https://civicdb.org/api/genes/1/comments]
#&gt;   Date: 2019-09-03 16:44
#&gt;   Status: 200
#&gt;   Content-Type: application/json; charset=utf-8
#&gt;   Size: 2 B
#&gt; 
#&gt; 
#&gt; attr(,"class")
#&gt; [1] "civic_api"</div><div class='input'><span class='fu'>getGeneMetadata</span>(<span class='kw'>id</span> <span class='kw'>=</span> <span class='fl'>1</span>, <span class='kw'>type</span> <span class='kw'>=</span> <span class='st'>"variants"</span>)</div><div class='output co'>#&gt; $content
#&gt; $content$`_meta`
#&gt; $content$`_meta`$current_page
#&gt; [1] 1
#&gt; 
#&gt; $content$`_meta`$per_page
#&gt; [1] 25
#&gt; 
#&gt; $content$`_meta`$total_pages
#&gt; [1] 2
#&gt; 
#&gt; $content$`_meta`$total_count
#&gt; [1] 46
#&gt; 
#&gt; $content$`_meta`$links
#&gt; $content$`_meta`$links$`next`
#&gt; [1] "https://civicdb.org/api/genes/1/variants?count=25&amp;page=2"
#&gt; 
#&gt; $content$`_meta`$links$previous
#&gt; NULL
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records
#&gt; $content$records[[1]]
#&gt; $content$records[[1]]$id
#&gt; [1] 5
#&gt; 
#&gt; $content$records[[1]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[1]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[1]]$name
#&gt; [1] "EML4-ALK"
#&gt; 
#&gt; $content$records[[1]]$description
#&gt; [1] "The EML4-ALK fusion variant 1 consisting of ALK kinase domain (exons 20-29) fused to EML4 exons 1-13 is the most common EML4-ALK variant, and was discovered in non-small cell lung cancer. Multiple EML4 breakpoint shave been described with differential sensitivity to inhibitors with variant 1 showing greater sensitivity than 3a in cell lines. EML4-ALK is crizotinib sensitive; however, several mutations that confer resistance mutations have been described in case studies. In the only clinical trial for crizotinib that included determination of EML4-ALK variant type in a subset of its participants, a very high response rate was observed, although the numbers were insufficient to validate correlation of variant type to outcome. Preclinical studies with this variant have indicated sensitivity to Hsp90 inhibitors."
#&gt; 
#&gt; $content$records[[1]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[1]]$variant_types
#&gt; $content$records[[1]]$variant_types[[1]]
#&gt; $content$records[[1]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[1]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[1]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[1]]$civic_actionability_score
#&gt; [1] 48
#&gt; 
#&gt; $content$records[[1]]$coordinates
#&gt; $content$records[[1]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[1]]$coordinates$start
#&gt; [1] 42396490
#&gt; 
#&gt; $content$records[[1]]$coordinates$stop
#&gt; [1] 42522656
#&gt; 
#&gt; $content$records[[1]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[1]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318522.5"
#&gt; 
#&gt; $content$records[[1]]$coordinates$chromosome2
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[1]]$coordinates$start2
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[1]]$coordinates$stop2
#&gt; [1] 29446394
#&gt; 
#&gt; $content$records[[1]]$coordinates$representative_transcript2
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[2]]
#&gt; $content$records[[2]]$id
#&gt; [1] 6
#&gt; 
#&gt; $content$records[[2]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[2]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[2]]$name
#&gt; [1] "EML4-ALK C1156Y"
#&gt; 
#&gt; $content$records[[2]]$description
#&gt; [1] "In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, C1156Y has been shown to confer resistance to crizotinib. Case reports indicate that secondary mutations can modulate drug sensitivity. EML4-ALK C1156Y/L1196M maintained crizotinib resistance while the lorlatinib resistant combination EML4-ALK C1156Y/L1198F re-sensitized the tumor to crizotinib treatment."
#&gt; 
#&gt; $content$records[[2]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[2]]$variant_types
#&gt; $content$records[[2]]$variant_types[[1]]
#&gt; $content$records[[2]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[2]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[2]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; $content$records[[2]]$variant_types[[2]]
#&gt; $content$records[[2]]$variant_types[[2]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[2]]$variant_types[[2]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[2]]$variant_types[[2]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[2]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[2]]$variant_types[[2]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[2]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[2]]$civic_actionability_score
#&gt; [1] 19
#&gt; 
#&gt; $content$records[[2]]$coordinates
#&gt; $content$records[[2]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[2]]$coordinates$start
#&gt; [1] 29445258
#&gt; 
#&gt; $content$records[[2]]$coordinates$stop
#&gt; [1] 29445258
#&gt; 
#&gt; $content$records[[2]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$records[[2]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$records[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[3]]
#&gt; $content$records[[3]]$id
#&gt; [1] 7
#&gt; 
#&gt; $content$records[[3]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[3]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[3]]$name
#&gt; [1] "EML4-ALK L1196M"
#&gt; 
#&gt; $content$records[[3]]$description
#&gt; [1] "In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, L1196M has been shown to confer resistance to crizotinib. This was also true in a patient with both EML4-ALK C1156Y &amp; L1196M."
#&gt; 
#&gt; $content$records[[3]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[3]]$variant_types
#&gt; $content$records[[3]]$variant_types[[1]]
#&gt; $content$records[[3]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[3]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[3]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; $content$records[[3]]$variant_types[[2]]
#&gt; $content$records[[3]]$variant_types[[2]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[3]]$variant_types[[2]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[3]]$variant_types[[2]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[3]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[3]]$variant_types[[2]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[3]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[3]]$civic_actionability_score
#&gt; [1] 31
#&gt; 
#&gt; $content$records[[3]]$coordinates
#&gt; $content$records[[3]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[3]]$coordinates$start
#&gt; [1] 29443631
#&gt; 
#&gt; $content$records[[3]]$coordinates$stop
#&gt; [1] 29443631
#&gt; 
#&gt; $content$records[[3]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$records[[3]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$records[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[4]]
#&gt; $content$records[[4]]$id
#&gt; [1] 8
#&gt; 
#&gt; $content$records[[4]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[4]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[4]]$name
#&gt; [1] "F1174L"
#&gt; 
#&gt; $content$records[[4]]$description
#&gt; [1] "ALK F1174L has been observed as recurrent in neuroblastoma, non-small cell lung cancer (NSCLC), and other cancer types. Neuroblastoma cells containing this mutation have shown resistance to low doses of criztonib. However, increased dosage can overcome this resistance in cell lines studies. TAE684 has also proven effective in both NSCLC and neuroblastoma F1174L containing cells."
#&gt; 
#&gt; $content$records[[4]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[4]]$variant_types
#&gt; $content$records[[4]]$variant_types[[1]]
#&gt; $content$records[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[4]]$civic_actionability_score
#&gt; [1] 33.5
#&gt; 
#&gt; $content$records[[4]]$coordinates
#&gt; $content$records[[4]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[4]]$coordinates$start
#&gt; [1] 29443695
#&gt; 
#&gt; $content$records[[4]]$coordinates$stop
#&gt; [1] 29443695
#&gt; 
#&gt; $content$records[[4]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$records[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$records[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[5]]
#&gt; $content$records[[5]]$id
#&gt; [1] 9
#&gt; 
#&gt; $content$records[[5]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[5]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[5]]$name
#&gt; [1] "R1275Q"
#&gt; 
#&gt; $content$records[[5]]$description
#&gt; [1] "ALK R1275Q has been observed as a recurrent mutation in  neuroblastoma, non-small cell lung cancer (NSCLC), as well as other cancer types. Neuroblastoma cells with this mutation have shown sensitivity to the ALK-inhibitor TAE684. This and the geldanamycin deriviative 17-DMAG have been shown to be effective in NSCLC cell lines."
#&gt; 
#&gt; $content$records[[5]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[5]]$variant_types
#&gt; $content$records[[5]]$variant_types[[1]]
#&gt; $content$records[[5]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[5]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[5]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[5]]$civic_actionability_score
#&gt; [1] 23
#&gt; 
#&gt; $content$records[[5]]$coordinates
#&gt; $content$records[[5]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[5]]$coordinates$start
#&gt; [1] 29432664
#&gt; 
#&gt; $content$records[[5]]$coordinates$stop
#&gt; [1] 29432664
#&gt; 
#&gt; $content$records[[5]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$records[[5]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$records[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[6]]
#&gt; $content$records[[6]]$id
#&gt; [1] 170
#&gt; 
#&gt; $content$records[[6]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[6]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[6]]$name
#&gt; [1] "EML4-ALK AMPLIFICATION"
#&gt; 
#&gt; $content$records[[6]]$description
#&gt; [1] ""
#&gt; 
#&gt; $content$records[[6]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[6]]$variant_types
#&gt; $content$records[[6]]$variant_types[[1]]
#&gt; $content$records[[6]]$variant_types[[1]]$id
#&gt; [1] 123
#&gt; 
#&gt; $content$records[[6]]$variant_types[[1]]$name
#&gt; [1] "transcript_amplification"
#&gt; 
#&gt; $content$records[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Amplification"
#&gt; 
#&gt; $content$records[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001889"
#&gt; 
#&gt; $content$records[[6]]$variant_types[[1]]$description
#&gt; [1] "A feature amplification of a region containing a transcript."
#&gt; 
#&gt; $content$records[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001889"
#&gt; 
#&gt; 
#&gt; $content$records[[6]]$variant_types[[2]]
#&gt; $content$records[[6]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[6]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[6]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[6]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[6]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[6]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[6]]$civic_actionability_score
#&gt; [1] 11
#&gt; 
#&gt; $content$records[[6]]$coordinates
#&gt; $content$records[[6]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[6]]$coordinates$start
#&gt; [1] 42396490
#&gt; 
#&gt; $content$records[[6]]$coordinates$stop
#&gt; [1] 42528380
#&gt; 
#&gt; $content$records[[6]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[6]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318522.5"
#&gt; 
#&gt; $content$records[[6]]$coordinates$chromosome2
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[6]]$coordinates$start2
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[6]]$coordinates$stop2
#&gt; [1] 29446394
#&gt; 
#&gt; $content$records[[6]]$coordinates$representative_transcript2
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[7]]
#&gt; $content$records[[7]]$id
#&gt; [1] 171
#&gt; 
#&gt; $content$records[[7]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[7]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[7]]$name
#&gt; [1] "ALK FUSION G1202R"
#&gt; 
#&gt; $content$records[[7]]$description
#&gt; [1] "ALK fusion with G1202R mutation in the ALK kinase domain was first identified in a lung adenocarcinoma patient who had progressed after 10 months of responding to crizotinib. Further characterization of this variant verified crizotinib resistance and additionally demonstrated resistance to multiple next generation ALK inhibitors including ceritinib, alectinib and brigatinib. Experiments using the HSP90 inhibitor  17-AAG have indicated potential sensitivity of EML4-ALK G1202R to this drug, suggesting this variant may be an HSP90 client protein."
#&gt; 
#&gt; $content$records[[7]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[7]]$variant_types
#&gt; $content$records[[7]]$variant_types[[1]]
#&gt; $content$records[[7]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[7]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[7]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; $content$records[[7]]$variant_types[[2]]
#&gt; $content$records[[7]]$variant_types[[2]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[7]]$variant_types[[2]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[7]]$variant_types[[2]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[7]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[7]]$variant_types[[2]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[7]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[7]]$civic_actionability_score
#&gt; [1] 23.5
#&gt; 
#&gt; $content$records[[7]]$coordinates
#&gt; $content$records[[7]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[7]]$coordinates$start
#&gt; [1] 29443613
#&gt; 
#&gt; $content$records[[7]]$coordinates$stop
#&gt; [1] 29443613
#&gt; 
#&gt; $content$records[[7]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$records[[7]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$records[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[8]]
#&gt; $content$records[[8]]$id
#&gt; [1] 172
#&gt; 
#&gt; $content$records[[8]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[8]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[8]]$name
#&gt; [1] "EML4-ALK S1206Y"
#&gt; 
#&gt; $content$records[[8]]$description
#&gt; [1] ""
#&gt; 
#&gt; $content$records[[8]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[8]]$variant_types
#&gt; $content$records[[8]]$variant_types[[1]]
#&gt; $content$records[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$records[[8]]$variant_types[[2]]
#&gt; $content$records[[8]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[8]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[8]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[8]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[8]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[8]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[8]]$civic_actionability_score
#&gt; [1] 6
#&gt; 
#&gt; $content$records[[8]]$coordinates
#&gt; $content$records[[8]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[8]]$coordinates$start
#&gt; [1] 29443600
#&gt; 
#&gt; $content$records[[8]]$coordinates$stop
#&gt; [1] 29443600
#&gt; 
#&gt; $content$records[[8]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$records[[8]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$records[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[9]]
#&gt; $content$records[[9]]$id
#&gt; [1] 173
#&gt; 
#&gt; $content$records[[9]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[9]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[9]]$name
#&gt; [1] "EML4-ALK T1151INST"
#&gt; 
#&gt; $content$records[[9]]$description
#&gt; [1] "The ALK fusion variant with insertion of T at amino acid 1151 was found in an ALK rearranged lung adenocarcinoma which had progressed after responding to crizotinib treatment. The mutation was absent in tumor sample taken before crizotinib treatment. Characterization of the variant using cell lines expressing EML4-ALK with 1151 T insert found the variant to be highly crizotinib resistant, and other work demonstrated resistance to next generation inhibitor alectinib, while resistance to tool compound TAE684 indicates potential resistance to the next generation ALK inhibitor ceritinib which is derived from TAE684. Other experiments suggest theraputic potential for HSP90 inhibitors, as EML4-ALK T1151 Tins sensitivity to 17-AAG has been observed."
#&gt; 
#&gt; $content$records[[9]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[9]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[9]]$variant_types
#&gt; $content$records[[9]]$variant_types[[1]]
#&gt; $content$records[[9]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[9]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[9]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[9]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[9]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[9]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; $content$records[[9]]$variant_types[[2]]
#&gt; $content$records[[9]]$variant_types[[2]]$id
#&gt; [1] 106
#&gt; 
#&gt; $content$records[[9]]$variant_types[[2]]$name
#&gt; [1] "inframe_insertion"
#&gt; 
#&gt; $content$records[[9]]$variant_types[[2]]$display_name
#&gt; [1] "Inframe Insertion"
#&gt; 
#&gt; $content$records[[9]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001821"
#&gt; 
#&gt; $content$records[[9]]$variant_types[[2]]$description
#&gt; [1] "An inframe non synonymous variant that inserts bases into in the coding sequence."
#&gt; 
#&gt; $content$records[[9]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001821"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[9]]$civic_actionability_score
#&gt; [1] 18
#&gt; 
#&gt; $content$records[[9]]$coordinates
#&gt; $content$records[[9]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[9]]$coordinates$start
#&gt; [1] 29445271
#&gt; 
#&gt; $content$records[[9]]$coordinates$stop
#&gt; [1] 29445272
#&gt; 
#&gt; $content$records[[9]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[9]]$coordinates$variant_bases
#&gt; [1] "CGT"
#&gt; 
#&gt; $content$records[[9]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[9]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[9]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[9]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[9]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[9]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[9]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[10]]
#&gt; $content$records[[10]]$id
#&gt; [1] 307
#&gt; 
#&gt; $content$records[[10]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[10]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[10]]$name
#&gt; [1] "EML4-ALK L1152R"
#&gt; 
#&gt; $content$records[[10]]$description
#&gt; [1] ""
#&gt; 
#&gt; $content$records[[10]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[10]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[10]]$variant_types
#&gt; $content$records[[10]]$variant_types[[1]]
#&gt; $content$records[[10]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[10]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[10]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[10]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[10]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[10]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$records[[10]]$variant_types[[2]]
#&gt; $content$records[[10]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[10]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[10]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[10]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[10]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[10]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[10]]$civic_actionability_score
#&gt; [1] 10
#&gt; 
#&gt; $content$records[[10]]$coordinates
#&gt; $content$records[[10]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[10]]$coordinates$start
#&gt; [1] 29445270
#&gt; 
#&gt; $content$records[[10]]$coordinates$stop
#&gt; [1] 29445270
#&gt; 
#&gt; $content$records[[10]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$records[[10]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$records[[10]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[10]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[10]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[10]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[10]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[10]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[10]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[11]]
#&gt; $content$records[[11]]$id
#&gt; [1] 308
#&gt; 
#&gt; $content$records[[11]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[11]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[11]]$name
#&gt; [1] "EML4-ALK G1269A"
#&gt; 
#&gt; $content$records[[11]]$description
#&gt; [1] "The EML4-ALK G1269A mutation has been identified in non-small cell lung cancer patients resistant to crizotinib. Cell line data from multiple groups have shown this mutation enhances resistance to crizotinib treatment in vitro."
#&gt; 
#&gt; $content$records[[11]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[11]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[11]]$variant_types
#&gt; $content$records[[11]]$variant_types[[1]]
#&gt; $content$records[[11]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[11]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[11]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[11]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[11]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[11]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; $content$records[[11]]$variant_types[[2]]
#&gt; $content$records[[11]]$variant_types[[2]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[11]]$variant_types[[2]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[11]]$variant_types[[2]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[11]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[11]]$variant_types[[2]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[11]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[11]]$civic_actionability_score
#&gt; [1] 17.5
#&gt; 
#&gt; $content$records[[11]]$coordinates
#&gt; $content$records[[11]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[11]]$coordinates$start
#&gt; [1] 29432682
#&gt; 
#&gt; $content$records[[11]]$coordinates$stop
#&gt; [1] 29432682
#&gt; 
#&gt; $content$records[[11]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$records[[11]]$coordinates$variant_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$records[[11]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[11]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[11]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[11]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[11]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[11]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[11]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[12]]
#&gt; $content$records[[12]]$id
#&gt; [1] 352
#&gt; 
#&gt; $content$records[[12]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[12]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[12]]$name
#&gt; [1] "EML4-ALK C1156Y–L1198F"
#&gt; 
#&gt; $content$records[[12]]$description
#&gt; [1] "EML4-ALK C1156Y alone is resistant to crizotinib in non-small cell lung cancer (NSCLC) which has been shown to be modulated (both increased sensitivity and resistance) by additional ALK mutations. A case report showed that an NSCLC harboring EML4-ALK C1156Y–L1198F was resistant to lorlatinib but this second mutation re-sensitized this tumor to crizotinib."
#&gt; 
#&gt; $content$records[[12]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[12]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[12]]$variant_types
#&gt; $content$records[[12]]$variant_types[[1]]
#&gt; $content$records[[12]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[12]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[12]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[12]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[12]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[12]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$records[[12]]$variant_types[[2]]
#&gt; $content$records[[12]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[12]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[12]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[12]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[12]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[12]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[12]]$civic_actionability_score
#&gt; [1] 20
#&gt; 
#&gt; $content$records[[12]]$coordinates
#&gt; $content$records[[12]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[12]]$coordinates$start
#&gt; [1] 29443625
#&gt; 
#&gt; $content$records[[12]]$coordinates$stop
#&gt; [1] 29445258
#&gt; 
#&gt; $content$records[[12]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[12]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[12]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[12]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[12]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[12]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[12]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[12]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[12]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[13]]
#&gt; $content$records[[13]]$id
#&gt; [1] 499
#&gt; 
#&gt; $content$records[[13]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[13]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[13]]$name
#&gt; [1] "ALK FUSIONS"
#&gt; 
#&gt; $content$records[[13]]$description
#&gt; [1] "Following the initial discovery and characterization of NPM-ALK fusion mutation in anaplastic large cell lymphoma (ALCL), EML4-ALK fusions were found to be driver mutations in a subset of lung adenocarcinomas. ALK fusions bring the C-terminal ALK kinase domain together with the N terminal region of the fusion partner, and have since been found in a range of cancers including inflammatory myofibroblastic tumor, renal cell carcinoma, diffuse large B cell lymphoma and colorectal cancer. \n\nIn a Phase I trial for crizotinib, a 92% response rate was seen in the subset of patients with a specific EML4-ALK variant, which was higher than the 57% rate in the overall ALK-rearranged population, but the subpopulation numbers were insufficient to correlate variant type with patient response. \n\nDue to restricted wild-type ALK expression, and also mild phenotypes seen in ALK knockout mice, ALK fusions are desirable targets.  ALK inhibitor TAE684 was used in initial characterizations of ALK fusion cancer drivers, and crizotinib – a small molecule ATP-competitor RTK inhibitor which is highly selective for ALK and MET - has been approved as first line therapy in ALK-rearranged NSCLC.   \n\nALK-rearrangement defines an NSCLC subtype that makes up 3-5% of NSCLC cases, and is characterized by higher proportions of younger patients and never smokers. EML4-ALK variant 1 is the most common ALK fusion in NSCLC and was the first to be discovered and demonstrate cancer driver properties in vitro and in mouse models. A Phase I study of crizotinib in ALK-rearranged NSCLC demonstrated an increase in overall survival when compared to standard chemotherapy. These and other initial results prompted accelerated approval for crizotinib in NSCLC.  Continued clinical work demonstrated benefit with crizotinib over chemotherapy in multi-armed studies, and as first line therapy. Crizotinib was also found to remain beneficial to NSCLC patients who had progressed on crizotinib. Additionally, crizotinib was shown to increase intracranial disease control in brain metastasis occurring from ALK-rearranged NSCLC."
#&gt; 
#&gt; $content$records[[13]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[13]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[13]]$variant_types
#&gt; $content$records[[13]]$variant_types[[1]]
#&gt; $content$records[[13]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[13]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[13]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[13]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[13]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[13]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[13]]$civic_actionability_score
#&gt; [1] 470.5
#&gt; 
#&gt; $content$records[[13]]$coordinates
#&gt; $content$records[[13]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[13]]$coordinates$start
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[13]]$coordinates$stop
#&gt; [1] 29446394
#&gt; 
#&gt; $content$records[[13]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[13]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[13]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[13]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[13]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[13]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[13]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[13]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[13]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[14]]
#&gt; $content$records[[14]]$id
#&gt; [1] 500
#&gt; 
#&gt; $content$records[[14]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[14]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[14]]$name
#&gt; [1] "EML4-ALK E20;A20"
#&gt; 
#&gt; $content$records[[14]]$description
#&gt; [1] "EML4-ALK variant 2 consists of a fusion of the ALK kinase domain (exons 20-29) with EML4 exons 1-20. Variant 2 has cytoplasmic distribution and makes up ~10% of EML4-ALK fusions found in NSCLC,. Although the specific variant is not consistently reported, single cases demonstrating response to crizotinib treatment in NSCLC and a differentiated malignant neoplasm of unknown origin have been reported. In vitro, this variant has shown greater sensitivity to ALK inhibition with TAE684 and crizotinib than the EML4-ALK variants 1 (E13;A20), 3a (E6;A20) and 3b (E6ins33;A20). The variant was reported to be less stable in cells than other variants, and may be a client protein of Hsp90. Along these lines the variant has shown preclinical sensitivity to Hsp90 inhibition."
#&gt; 
#&gt; $content$records[[14]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[14]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[14]]$variant_types
#&gt; $content$records[[14]]$variant_types[[1]]
#&gt; $content$records[[14]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[14]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[14]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[14]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[14]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[14]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[14]]$civic_actionability_score
#&gt; [1] 16.5
#&gt; 
#&gt; $content$records[[14]]$coordinates
#&gt; $content$records[[14]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[14]]$coordinates$start
#&gt; [1] 42396490
#&gt; 
#&gt; $content$records[[14]]$coordinates$stop
#&gt; [1] 42552694
#&gt; 
#&gt; $content$records[[14]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[14]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[14]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318522.5"
#&gt; 
#&gt; $content$records[[14]]$coordinates$chromosome2
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[14]]$coordinates$start2
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[14]]$coordinates$stop2
#&gt; [1] 29446394
#&gt; 
#&gt; $content$records[[14]]$coordinates$representative_transcript2
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[14]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[14]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[15]]
#&gt; $content$records[[15]]$id
#&gt; [1] 501
#&gt; 
#&gt; $content$records[[15]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[15]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[15]]$name
#&gt; [1] "EML4-ALK E2;A20"
#&gt; 
#&gt; $content$records[[15]]$description
#&gt; [1] ""
#&gt; 
#&gt; $content$records[[15]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[15]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[15]]$variant_types
#&gt; $content$records[[15]]$variant_types[[1]]
#&gt; $content$records[[15]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[15]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[15]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[15]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[15]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[15]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[15]]$civic_actionability_score
#&gt; [1] 1.75
#&gt; 
#&gt; $content$records[[15]]$coordinates
#&gt; $content$records[[15]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[15]]$coordinates$start
#&gt; [1] 42396490
#&gt; 
#&gt; $content$records[[15]]$coordinates$stop
#&gt; [1] 42472827
#&gt; 
#&gt; $content$records[[15]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[15]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[15]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318522.5"
#&gt; 
#&gt; $content$records[[15]]$coordinates$chromosome2
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[15]]$coordinates$start2
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[15]]$coordinates$stop2
#&gt; [1] 29446394
#&gt; 
#&gt; $content$records[[15]]$coordinates$representative_transcript2
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[15]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[15]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[16]]
#&gt; $content$records[[16]]$id
#&gt; [1] 503
#&gt; 
#&gt; $content$records[[16]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[16]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[16]]$name
#&gt; [1] "EML4-ALK E6;A20"
#&gt; 
#&gt; $content$records[[16]]$description
#&gt; [1] "The EML4-ALK variant 3a consists of the ALK kinase domain fused to EML4 exons 1-6, while the 3b variant has an additional 33bp of intronic EML4 sequence between EML4 and ALK regions. The two variants were found to originate from a single genomic rearrangement, and 3a/3b is found in ~ 30% of EML4-ALK positive NSCLC. Variant 3a has cytoplasmic and nuclear distribution. In vitro, these variants have shown sensitivity to ALK inhibitors TAE684 and crizotinib but less sensitive than other EML4-ALK variants. The H2228 cell line contains 3a and 3b variants and is described as having cyctostatic response to ALK inhibition as opposed to apoptotic. In a study of 31 NSCLC patients with EML4-ALK rearrangements, 4/5 patients with the 3a variant showed a partial response while 1 showed progressive disease (the only PD observed). In cells the 3a variant was reported to be more stable than other variants, less sensitive to crizotinib and not to respond to Hsp90 inhibition."
#&gt; 
#&gt; $content$records[[16]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[16]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[16]]$variant_types
#&gt; $content$records[[16]]$variant_types[[1]]
#&gt; $content$records[[16]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[16]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[16]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[16]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[16]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[16]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[16]]$civic_actionability_score
#&gt; [1] 25
#&gt; 
#&gt; $content$records[[16]]$coordinates
#&gt; $content$records[[16]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[16]]$coordinates$start
#&gt; [1] 42396490
#&gt; 
#&gt; $content$records[[16]]$coordinates$stop
#&gt; [1] 42491871
#&gt; 
#&gt; $content$records[[16]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[16]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[16]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318522.5"
#&gt; 
#&gt; $content$records[[16]]$coordinates$chromosome2
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[16]]$coordinates$start2
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[16]]$coordinates$stop2
#&gt; [1] 29446394
#&gt; 
#&gt; $content$records[[16]]$coordinates$representative_transcript2
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[16]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[16]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[17]]
#&gt; $content$records[[17]]$id
#&gt; [1] 512
#&gt; 
#&gt; $content$records[[17]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[17]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[17]]$name
#&gt; [1] "MUTATION"
#&gt; 
#&gt; $content$records[[17]]$description
#&gt; [1] ""
#&gt; 
#&gt; $content$records[[17]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[17]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[17]]$variant_types
#&gt; $content$records[[17]]$variant_types[[1]]
#&gt; $content$records[[17]]$variant_types[[1]]$id
#&gt; [1] 30
#&gt; 
#&gt; $content$records[[17]]$variant_types[[1]]$name
#&gt; [1] "gene_variant"
#&gt; 
#&gt; $content$records[[17]]$variant_types[[1]]$display_name
#&gt; [1] "Gene Variant"
#&gt; 
#&gt; $content$records[[17]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001564"
#&gt; 
#&gt; $content$records[[17]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant where the structure of the gene is changed."
#&gt; 
#&gt; $content$records[[17]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001564"
#&gt; 
#&gt; 
#&gt; $content$records[[17]]$variant_types[[2]]
#&gt; $content$records[[17]]$variant_types[[2]]$id
#&gt; [1] 160
#&gt; 
#&gt; $content$records[[17]]$variant_types[[2]]$name
#&gt; [1] "gain_of_function_variant"
#&gt; 
#&gt; $content$records[[17]]$variant_types[[2]]$display_name
#&gt; [1] "Gain Of Function Variant"
#&gt; 
#&gt; $content$records[[17]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0002053"
#&gt; 
#&gt; $content$records[[17]]$variant_types[[2]]$description
#&gt; [1] "A sequence variant whereby new or enhanced function is conferred on the gene product."
#&gt; 
#&gt; $content$records[[17]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0002053"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[17]]$civic_actionability_score
#&gt; [1] 20
#&gt; 
#&gt; $content$records[[17]]$coordinates
#&gt; $content$records[[17]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[17]]$coordinates$start
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[17]]$coordinates$stop
#&gt; [1] 30144432
#&gt; 
#&gt; $content$records[[17]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[17]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[17]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[17]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[17]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[17]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[17]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[17]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[17]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[18]]
#&gt; $content$records[[18]]$id
#&gt; [1] 513
#&gt; 
#&gt; $content$records[[18]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[18]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[18]]$name
#&gt; [1] "NPM-ALK"
#&gt; 
#&gt; $content$records[[18]]$description
#&gt; [1] ""
#&gt; 
#&gt; $content$records[[18]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[18]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[18]]$variant_types
#&gt; $content$records[[18]]$variant_types[[1]]
#&gt; $content$records[[18]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[18]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[18]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[18]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[18]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[18]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[18]]$civic_actionability_score
#&gt; [1] 27
#&gt; 
#&gt; $content$records[[18]]$coordinates
#&gt; $content$records[[18]]$coordinates$chromosome
#&gt; [1] "5"
#&gt; 
#&gt; $content$records[[18]]$coordinates$start
#&gt; [1] 170814120
#&gt; 
#&gt; $content$records[[18]]$coordinates$stop
#&gt; [1] 170818803
#&gt; 
#&gt; $content$records[[18]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[18]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[18]]$coordinates$representative_transcript
#&gt; [1] "ENST00000517671.1"
#&gt; 
#&gt; $content$records[[18]]$coordinates$chromosome2
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[18]]$coordinates$start2
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[18]]$coordinates$stop2
#&gt; [1] 29446394
#&gt; 
#&gt; $content$records[[18]]$coordinates$representative_transcript2
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[18]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[18]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[19]]
#&gt; $content$records[[19]]$id
#&gt; [1] 514
#&gt; 
#&gt; $content$records[[19]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[19]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[19]]$name
#&gt; [1] "RANBP2-ALK"
#&gt; 
#&gt; $content$records[[19]]$description
#&gt; [1] ""
#&gt; 
#&gt; $content$records[[19]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[19]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[19]]$variant_types
#&gt; $content$records[[19]]$variant_types[[1]]
#&gt; $content$records[[19]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[19]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[19]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[19]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[19]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[19]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[19]]$civic_actionability_score
#&gt; [1] 10
#&gt; 
#&gt; $content$records[[19]]$coordinates
#&gt; $content$records[[19]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[19]]$coordinates$start
#&gt; [1] 109335937
#&gt; 
#&gt; $content$records[[19]]$coordinates$stop
#&gt; [1] 109375004
#&gt; 
#&gt; $content$records[[19]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[19]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[19]]$coordinates$representative_transcript
#&gt; [1] "ENST00000283195.6"
#&gt; 
#&gt; $content$records[[19]]$coordinates$chromosome2
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[19]]$coordinates$start2
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[19]]$coordinates$stop2
#&gt; [1] 29446394
#&gt; 
#&gt; $content$records[[19]]$coordinates$representative_transcript2
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[19]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[19]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[20]]
#&gt; $content$records[[20]]$id
#&gt; [1] 520
#&gt; 
#&gt; $content$records[[20]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[20]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[20]]$name
#&gt; [1] "CLTC-ALK"
#&gt; 
#&gt; $content$records[[20]]$description
#&gt; [1] "The t(2;17)(p23;q23) translocation results in the CLTC-ALK fusion protein, the most common ALK fusion observed in diffuse large B cell lymphoma (DLBCL). ALK-rearranged DLBCL is less responsive to CHOP chemotherapy. Preclinical work indicates that CLTC-ALK DLBCL is responsive to ALK inhibition in cell lines and mouse models, and two case studies show short response followed by progression when heavily pretreated and advanced ALK-positive DLBCL is treated with crizotinib (one case CLTC-ALK, one case an unknown ALK fusion)."
#&gt; 
#&gt; $content$records[[20]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[20]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[20]]$variant_types
#&gt; $content$records[[20]]$variant_types[[1]]
#&gt; $content$records[[20]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[20]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[20]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[20]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[20]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[20]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[20]]$civic_actionability_score
#&gt; [1] 19
#&gt; 
#&gt; $content$records[[20]]$coordinates
#&gt; $content$records[[20]]$coordinates$chromosome
#&gt; [1] "17"
#&gt; 
#&gt; $content$records[[20]]$coordinates$start
#&gt; [1] 57697219
#&gt; 
#&gt; $content$records[[20]]$coordinates$stop
#&gt; [1] 57768072
#&gt; 
#&gt; $content$records[[20]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[20]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[20]]$coordinates$representative_transcript
#&gt; [1] "ENST00000269122.3"
#&gt; 
#&gt; $content$records[[20]]$coordinates$chromosome2
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[20]]$coordinates$start2
#&gt; [1] 29415640
#&gt; 
#&gt; $content$records[[20]]$coordinates$stop2
#&gt; [1] 29446394
#&gt; 
#&gt; $content$records[[20]]$coordinates$representative_transcript2
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[20]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[20]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[21]]
#&gt; $content$records[[21]]$id
#&gt; [1] 527
#&gt; 
#&gt; $content$records[[21]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[21]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[21]]$name
#&gt; [1] "ALK FUSION I1171"
#&gt; 
#&gt; $content$records[[21]]$description
#&gt; [1] "Multiple case reports indicate that the I1171T missense mutation in ALK-rearranged non-small cell lung cancer (NSCLC) confers resistance to crizotinib and alectinib but is sensitive to ceritinib treatment. These results have been validated in vitro. In vitro evidence indicates this mutation interferes with alectinib binding the active site of ALK and may explain the sensitivity to the structurally distinct ALK inhibitor ceritinib."
#&gt; 
#&gt; $content$records[[21]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[21]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[21]]$variant_types
#&gt; $content$records[[21]]$variant_types[[1]]
#&gt; $content$records[[21]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[21]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[21]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[21]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[21]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[21]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; $content$records[[21]]$variant_types[[2]]
#&gt; $content$records[[21]]$variant_types[[2]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[21]]$variant_types[[2]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[21]]$variant_types[[2]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[21]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[21]]$variant_types[[2]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[21]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[21]]$civic_actionability_score
#&gt; [1] 42.5
#&gt; 
#&gt; $content$records[[21]]$coordinates
#&gt; $content$records[[21]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[21]]$coordinates$start
#&gt; [1] 29445213
#&gt; 
#&gt; $content$records[[21]]$coordinates$stop
#&gt; [1] 29445213
#&gt; 
#&gt; $content$records[[21]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$records[[21]]$coordinates$variant_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$records[[21]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[21]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[21]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[21]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[21]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[21]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[21]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[22]]
#&gt; $content$records[[22]]$id
#&gt; [1] 528
#&gt; 
#&gt; $content$records[[22]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[22]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[22]]$name
#&gt; [1] "EML4-ALK  V1180L"
#&gt; 
#&gt; $content$records[[22]]$description
#&gt; [1] "The V1180L missense mutation was identified in a mutagenesis screen to find mutations that conferred resistance to alectinib in a non-small cell lung cancer cell line. This mutation shows resistance to crizotinib and alectinib but sensitivity to TAE684, ceritinib and more mild sensitivity to brigatinib. Similar to I1171T, it resides in the ATP pocket of ALK and likely interferes with alectinib and crizotinib binding as a mechanism of resistance."
#&gt; 
#&gt; $content$records[[22]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[22]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[22]]$variant_types
#&gt; $content$records[[22]]$variant_types[[1]]
#&gt; $content$records[[22]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[22]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[22]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[22]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[22]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[22]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$records[[22]]$variant_types[[2]]
#&gt; $content$records[[22]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[22]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[22]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[22]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[22]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[22]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[22]]$civic_actionability_score
#&gt; [1] 13
#&gt; 
#&gt; $content$records[[22]]$coordinates
#&gt; $content$records[[22]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[22]]$coordinates$start
#&gt; [1] 29443679
#&gt; 
#&gt; $content$records[[22]]$coordinates$stop
#&gt; [1] 29443679
#&gt; 
#&gt; $content$records[[22]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$records[[22]]$coordinates$variant_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$records[[22]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[22]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[22]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[22]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[22]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[22]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[22]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[23]]
#&gt; $content$records[[23]]$id
#&gt; [1] 549
#&gt; 
#&gt; $content$records[[23]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[23]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[23]]$name
#&gt; [1] "F1245C"
#&gt; 
#&gt; $content$records[[23]]$description
#&gt; [1] ""
#&gt; 
#&gt; $content$records[[23]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[23]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[23]]$variant_types
#&gt; $content$records[[23]]$variant_types[[1]]
#&gt; $content$records[[23]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[23]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[23]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[23]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[23]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[23]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[23]]$civic_actionability_score
#&gt; [1] 10.5
#&gt; 
#&gt; $content$records[[23]]$coordinates
#&gt; $content$records[[23]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[23]]$coordinates$start
#&gt; [1] 29436859
#&gt; 
#&gt; $content$records[[23]]$coordinates$stop
#&gt; [1] 29436859
#&gt; 
#&gt; $content$records[[23]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$records[[23]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$records[[23]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[23]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[23]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[23]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[23]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[23]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[23]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[24]]
#&gt; $content$records[[24]]$id
#&gt; [1] 550
#&gt; 
#&gt; $content$records[[24]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[24]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[24]]$name
#&gt; [1] "DEL4-11"
#&gt; 
#&gt; $content$records[[24]]$description
#&gt; [1] ""
#&gt; 
#&gt; $content$records[[24]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[24]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[24]]$variant_types
#&gt; $content$records[[24]]$variant_types[[1]]
#&gt; $content$records[[24]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$records[[24]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$records[[24]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$records[[24]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$records[[24]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$records[[24]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[24]]$civic_actionability_score
#&gt; [1] 3
#&gt; 
#&gt; $content$records[[24]]$coordinates
#&gt; $content$records[[24]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[24]]$coordinates$start
#&gt; [1] 29497965
#&gt; 
#&gt; $content$records[[24]]$coordinates$stop
#&gt; [1] 29754982
#&gt; 
#&gt; $content$records[[24]]$coordinates$reference_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[24]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$records[[24]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[24]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[24]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[24]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[24]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[24]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[24]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[25]]
#&gt; $content$records[[25]]$id
#&gt; [1] 551
#&gt; 
#&gt; $content$records[[25]]$entrez_name
#&gt; [1] "ALK"
#&gt; 
#&gt; $content$records[[25]]$entrez_id
#&gt; [1] 238
#&gt; 
#&gt; $content$records[[25]]$name
#&gt; [1] "ALK FUSION F1245C"
#&gt; 
#&gt; $content$records[[25]]$description
#&gt; [1] "In neuroblastoma, F1245 is one of three hotspot ALK mutations found at a high frequency. In a case study, a pretreated non-small cell lung cancer (NSCLC) patient with EML4-ALK F1245C mutation responded well to ceritinib after slow disease progression during crizotinib treatment. Crizotinib resistance was attributed to the F1245C mutation, although the ALK-rearrangement was not sequenced prior to crizotinib treatment. The patient's slow onset of disease progression during crizotinib treatment mirrors preclinical findings that indicate ALK-fusion with F1245C has moderate resistance to crizotinib compared to the fusion alone."
#&gt; 
#&gt; $content$records[[25]]$gene_id
#&gt; [1] 1
#&gt; 
#&gt; $content$records[[25]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$records[[25]]$variant_types
#&gt; $content$records[[25]]$variant_types[[1]]
#&gt; $content$records[[25]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$records[[25]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$records[[25]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$records[[25]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$records[[25]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$records[[25]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$records[[25]]$variant_types[[2]]
#&gt; $content$records[[25]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$records[[25]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$records[[25]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$records[[25]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$records[[25]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$records[[25]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$records[[25]]$civic_actionability_score
#&gt; [1] 14.5
#&gt; 
#&gt; $content$records[[25]]$coordinates
#&gt; $content$records[[25]]$coordinates$chromosome
#&gt; [1] "2"
#&gt; 
#&gt; $content$records[[25]]$coordinates$start
#&gt; [1] 29436859
#&gt; 
#&gt; $content$records[[25]]$coordinates$stop
#&gt; [1] 29436859
#&gt; 
#&gt; $content$records[[25]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$records[[25]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$records[[25]]$coordinates$representative_transcript
#&gt; [1] "ENST00000389048.3"
#&gt; 
#&gt; $content$records[[25]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$records[[25]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$records[[25]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$records[[25]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$records[[25]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$records[[25]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $url
#&gt; [1] "https://civicdb.org/api/genes/1/variants"
#&gt; 
#&gt; $response
#&gt; Response [https://civicdb.org/api/genes/1/variants]
#&gt;   Date: 2019-09-03 16:44
#&gt;   Status: 200
#&gt;   Content-Type: application/json; charset=utf-8
#&gt;   Size: 32.1 kB
#&gt; 
#&gt; 
#&gt; attr(,"class")
#&gt; [1] "civic_api"</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
    <h2>Contents</h2>
    <ul class="nav nav-pills nav-stacked">
      <li><a href="#arguments">Arguments</a></li>
      
      <li><a href="#value">Value</a></li>
      
      <li><a href="#examples">Examples</a></li>
    </ul>

  </div>
</div>

      <footer>
      <div class="copyright">
  <p>Developed by Andrew Duncan.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
   </div>

  

  </body>
</html>

