<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Get a specific variant group — getVariantGroup • civicr</title>

<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha256-916EbMg70RQy9LHiGkXzG8hSg9EdNy97GazNG/aiY1w=" crossorigin="anonymous" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script>

<!-- sticky kit -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>



<meta property="og:title" content="Get a specific variant group — getVariantGroup" />

<meta property="og:description" content="Retrieve a specific variant group from the CIViC DB" />
<meta name="twitter:card" content="summary" />



<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->


  </head>

  <body>
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">civicr</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fa fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
      
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      
      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Get a specific variant group</h1>
    
    <div class="hidden name"><code>getVariantGroup.Rd</code></div>
    </div>

    <div class="ref-description">
    
    <p>Retrieve a specific variant group from the CIViC DB</p>
    
    </div>

    <pre class="usage"><span class='fu'>getVariantGroup</span>(<span class='no'>id</span>)</pre>
    
    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>id</th>
      <td><p>Internal CIViC ID of the variant group of interest</p></td>
    </tr>
    </table>
    
    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>An S3 Object of type civic_api containing the content, url, and response</p>
    

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='fu'>getVariantGroup</span>(<span class='kw'>id</span> <span class='kw'>=</span> <span class='fl'>1</span>)</div><div class='output co'>#&gt; $content
#&gt; $content$id
#&gt; [1] 1
#&gt; 
#&gt; $content$name
#&gt; [1] "Imatinib Resistance"
#&gt; 
#&gt; $content$description
#&gt; [1] "While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "
#&gt; 
#&gt; $content$variants
#&gt; $content$variants[[1]]
#&gt; $content$variants[[1]]$id
#&gt; [1] 2
#&gt; 
#&gt; $content$variants[[1]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[1]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[1]]$name
#&gt; [1] "BCR-ABL T315I"
#&gt; 
#&gt; $content$variants[[1]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[1]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_types
#&gt; $content$variants[[1]]$variant_types[[1]]
#&gt; $content$variants[[1]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[2]]
#&gt; $content$variants[[1]]$variant_types[[2]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[2]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[2]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[2]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[1]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$civic_actionability_score
#&gt; [1] 105
#&gt; 
#&gt; $content$variants[[1]]$coordinates
#&gt; $content$variants[[1]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[1]]$coordinates$start
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[1]]$coordinates$stop
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[1]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[1]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[1]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups
#&gt; $content$variants[[1]]$variant_groups[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$id
#&gt; [1] 1
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$name
#&gt; [1] "Imatinib Resistance"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$description
#&gt; [1] "While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$id
#&gt; [1] 2
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$name
#&gt; [1] "BCR-ABL T315I"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$civic_actionability_score
#&gt; [1] 105
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$start
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$stop
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$id
#&gt; [1] 3
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$name
#&gt; [1] "BCR-ABL E255K"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$civic_actionability_score
#&gt; [1] 56
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$start
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$stop
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$variant_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$id
#&gt; [1] 98
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$name
#&gt; [1] "D842I"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$variant_types
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_bases
#&gt; [1] "GA"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$variant_bases
#&gt; [1] "AT"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$id
#&gt; [1] 99
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$name
#&gt; [1] "D842V"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$variant_types
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$civic_actionability_score
#&gt; [1] 100.5
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$id
#&gt; [1] 100
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$name
#&gt; [1] "D842Y"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$variant_types
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$stop
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$id
#&gt; [1] 101
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$name
#&gt; [1] "I843DEL"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$variant_types
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$civic_actionability_score
#&gt; [1] 5
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$start
#&gt; [1] 55152095
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_bases
#&gt; [1] "ATC"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$id
#&gt; [1] 102
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$name
#&gt; [1] "DI842-843VM"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$variant_types
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_bases
#&gt; [1] "ACATC"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$variant_bases
#&gt; [1] "TCATG"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$id
#&gt; [1] 241
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$name
#&gt; [1] "BCR-ABL F317L"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$description
#&gt; [1] "BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$civic_actionability_score
#&gt; [1] 99.5
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$start
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$stop
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[1]]$variant_groups[[1]]$type
#&gt; [1] "variant_group"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]
#&gt; $content$variants[[2]]$id
#&gt; [1] 102
#&gt; 
#&gt; $content$variants[[2]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[2]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[2]]$name
#&gt; [1] "DI842-843VM"
#&gt; 
#&gt; $content$variants[[2]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[2]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_types
#&gt; $content$variants[[2]]$variant_types[[1]]
#&gt; $content$variants[[2]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[2]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[2]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[2]]$coordinates
#&gt; $content$variants[[2]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[2]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[2]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[2]]$coordinates$reference_bases
#&gt; [1] "ACATC"
#&gt; 
#&gt; $content$variants[[2]]$coordinates$variant_bases
#&gt; [1] "TCATG"
#&gt; 
#&gt; $content$variants[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups
#&gt; $content$variants[[2]]$variant_groups[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$id
#&gt; [1] 1
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$name
#&gt; [1] "Imatinib Resistance"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$description
#&gt; [1] "While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$id
#&gt; [1] 2
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$name
#&gt; [1] "BCR-ABL T315I"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$civic_actionability_score
#&gt; [1] 105
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$start
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$stop
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$id
#&gt; [1] 3
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$name
#&gt; [1] "BCR-ABL E255K"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$civic_actionability_score
#&gt; [1] 56
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$start
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$stop
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$variant_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$id
#&gt; [1] 98
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$name
#&gt; [1] "D842I"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$variant_types
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_bases
#&gt; [1] "GA"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$variant_bases
#&gt; [1] "AT"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$id
#&gt; [1] 99
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$name
#&gt; [1] "D842V"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$variant_types
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$civic_actionability_score
#&gt; [1] 100.5
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$id
#&gt; [1] 100
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$name
#&gt; [1] "D842Y"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$variant_types
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$stop
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$id
#&gt; [1] 101
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$name
#&gt; [1] "I843DEL"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$variant_types
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$civic_actionability_score
#&gt; [1] 5
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$start
#&gt; [1] 55152095
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_bases
#&gt; [1] "ATC"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$id
#&gt; [1] 102
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$name
#&gt; [1] "DI842-843VM"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$variant_types
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_bases
#&gt; [1] "ACATC"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$variant_bases
#&gt; [1] "TCATG"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$id
#&gt; [1] 241
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$name
#&gt; [1] "BCR-ABL F317L"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$description
#&gt; [1] "BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$civic_actionability_score
#&gt; [1] 99.5
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$start
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$stop
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[2]]$variant_groups[[1]]$type
#&gt; [1] "variant_group"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]
#&gt; $content$variants[[3]]$id
#&gt; [1] 98
#&gt; 
#&gt; $content$variants[[3]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[3]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[3]]$name
#&gt; [1] "D842I"
#&gt; 
#&gt; $content$variants[[3]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[3]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_types
#&gt; $content$variants[[3]]$variant_types[[1]]
#&gt; $content$variants[[3]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[3]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[3]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[3]]$coordinates
#&gt; $content$variants[[3]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[3]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[3]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[3]]$coordinates$reference_bases
#&gt; [1] "GA"
#&gt; 
#&gt; $content$variants[[3]]$coordinates$variant_bases
#&gt; [1] "AT"
#&gt; 
#&gt; $content$variants[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups
#&gt; $content$variants[[3]]$variant_groups[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$id
#&gt; [1] 1
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$name
#&gt; [1] "Imatinib Resistance"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$description
#&gt; [1] "While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$id
#&gt; [1] 2
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$name
#&gt; [1] "BCR-ABL T315I"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$civic_actionability_score
#&gt; [1] 105
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$start
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$stop
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$id
#&gt; [1] 3
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$name
#&gt; [1] "BCR-ABL E255K"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$civic_actionability_score
#&gt; [1] 56
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$start
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$stop
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$variant_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$id
#&gt; [1] 98
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$name
#&gt; [1] "D842I"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$variant_types
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_bases
#&gt; [1] "GA"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$variant_bases
#&gt; [1] "AT"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$id
#&gt; [1] 99
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$name
#&gt; [1] "D842V"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$variant_types
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$civic_actionability_score
#&gt; [1] 100.5
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$id
#&gt; [1] 100
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$name
#&gt; [1] "D842Y"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$variant_types
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$stop
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$id
#&gt; [1] 101
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$name
#&gt; [1] "I843DEL"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$variant_types
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$civic_actionability_score
#&gt; [1] 5
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$start
#&gt; [1] 55152095
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_bases
#&gt; [1] "ATC"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$id
#&gt; [1] 102
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$name
#&gt; [1] "DI842-843VM"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$variant_types
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_bases
#&gt; [1] "ACATC"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$variant_bases
#&gt; [1] "TCATG"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$id
#&gt; [1] 241
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$name
#&gt; [1] "BCR-ABL F317L"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$description
#&gt; [1] "BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$civic_actionability_score
#&gt; [1] 99.5
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$start
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$stop
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[3]]$variant_groups[[1]]$type
#&gt; [1] "variant_group"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]
#&gt; $content$variants[[4]]$id
#&gt; [1] 100
#&gt; 
#&gt; $content$variants[[4]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[4]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[4]]$name
#&gt; [1] "D842Y"
#&gt; 
#&gt; $content$variants[[4]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[4]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_types
#&gt; $content$variants[[4]]$variant_types[[1]]
#&gt; $content$variants[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[4]]$coordinates
#&gt; $content$variants[[4]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[4]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[4]]$coordinates$stop
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[4]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups
#&gt; $content$variants[[4]]$variant_groups[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$id
#&gt; [1] 1
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$name
#&gt; [1] "Imatinib Resistance"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$description
#&gt; [1] "While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$id
#&gt; [1] 2
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$name
#&gt; [1] "BCR-ABL T315I"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$civic_actionability_score
#&gt; [1] 105
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$start
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$stop
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$id
#&gt; [1] 3
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$name
#&gt; [1] "BCR-ABL E255K"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$civic_actionability_score
#&gt; [1] 56
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$start
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$stop
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$variant_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$id
#&gt; [1] 98
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$name
#&gt; [1] "D842I"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$variant_types
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_bases
#&gt; [1] "GA"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$variant_bases
#&gt; [1] "AT"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$id
#&gt; [1] 99
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$name
#&gt; [1] "D842V"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$variant_types
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$civic_actionability_score
#&gt; [1] 100.5
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$id
#&gt; [1] 100
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$name
#&gt; [1] "D842Y"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$variant_types
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$stop
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$id
#&gt; [1] 101
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$name
#&gt; [1] "I843DEL"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$variant_types
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$civic_actionability_score
#&gt; [1] 5
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$start
#&gt; [1] 55152095
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_bases
#&gt; [1] "ATC"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$id
#&gt; [1] 102
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$name
#&gt; [1] "DI842-843VM"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$variant_types
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_bases
#&gt; [1] "ACATC"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$variant_bases
#&gt; [1] "TCATG"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$id
#&gt; [1] 241
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$name
#&gt; [1] "BCR-ABL F317L"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$description
#&gt; [1] "BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$civic_actionability_score
#&gt; [1] 99.5
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$start
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$stop
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[4]]$variant_groups[[1]]$type
#&gt; [1] "variant_group"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]
#&gt; $content$variants[[5]]$id
#&gt; [1] 241
#&gt; 
#&gt; $content$variants[[5]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[5]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[5]]$name
#&gt; [1] "BCR-ABL F317L"
#&gt; 
#&gt; $content$variants[[5]]$description
#&gt; [1] "BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."
#&gt; 
#&gt; $content$variants[[5]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_types
#&gt; $content$variants[[5]]$variant_types[[1]]
#&gt; $content$variants[[5]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[2]]
#&gt; $content$variants[[5]]$variant_types[[2]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[2]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[2]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[2]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[5]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$civic_actionability_score
#&gt; [1] 99.5
#&gt; 
#&gt; $content$variants[[5]]$coordinates
#&gt; $content$variants[[5]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[5]]$coordinates$start
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[5]]$coordinates$stop
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[5]]$coordinates$reference_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[5]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups
#&gt; $content$variants[[5]]$variant_groups[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$id
#&gt; [1] 1
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$name
#&gt; [1] "Imatinib Resistance"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$description
#&gt; [1] "While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$id
#&gt; [1] 2
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$name
#&gt; [1] "BCR-ABL T315I"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$civic_actionability_score
#&gt; [1] 105
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$start
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$stop
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$id
#&gt; [1] 3
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$name
#&gt; [1] "BCR-ABL E255K"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$civic_actionability_score
#&gt; [1] 56
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$start
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$stop
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$variant_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$id
#&gt; [1] 98
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$name
#&gt; [1] "D842I"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$variant_types
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_bases
#&gt; [1] "GA"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$variant_bases
#&gt; [1] "AT"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$id
#&gt; [1] 99
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$name
#&gt; [1] "D842V"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$variant_types
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$civic_actionability_score
#&gt; [1] 100.5
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$id
#&gt; [1] 100
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$name
#&gt; [1] "D842Y"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$variant_types
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$stop
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$id
#&gt; [1] 101
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$name
#&gt; [1] "I843DEL"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$variant_types
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$civic_actionability_score
#&gt; [1] 5
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$start
#&gt; [1] 55152095
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_bases
#&gt; [1] "ATC"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$id
#&gt; [1] 102
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$name
#&gt; [1] "DI842-843VM"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$variant_types
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_bases
#&gt; [1] "ACATC"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$variant_bases
#&gt; [1] "TCATG"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$id
#&gt; [1] 241
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$name
#&gt; [1] "BCR-ABL F317L"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$description
#&gt; [1] "BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$civic_actionability_score
#&gt; [1] 99.5
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$start
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$stop
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[5]]$variant_groups[[1]]$type
#&gt; [1] "variant_group"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]
#&gt; $content$variants[[6]]$id
#&gt; [1] 3
#&gt; 
#&gt; $content$variants[[6]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[6]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[6]]$name
#&gt; [1] "BCR-ABL E255K"
#&gt; 
#&gt; $content$variants[[6]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[6]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_types
#&gt; $content$variants[[6]]$variant_types[[1]]
#&gt; $content$variants[[6]]$variant_types[[1]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[1]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[1]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[2]]
#&gt; $content$variants[[6]]$variant_types[[2]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[2]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[2]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[2]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[6]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$civic_actionability_score
#&gt; [1] 56
#&gt; 
#&gt; $content$variants[[6]]$coordinates
#&gt; $content$variants[[6]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[6]]$coordinates$start
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[6]]$coordinates$stop
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[6]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[6]]$coordinates$variant_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[6]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups
#&gt; $content$variants[[6]]$variant_groups[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$id
#&gt; [1] 1
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$name
#&gt; [1] "Imatinib Resistance"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$description
#&gt; [1] "While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$id
#&gt; [1] 2
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$name
#&gt; [1] "BCR-ABL T315I"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$civic_actionability_score
#&gt; [1] 105
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$start
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$stop
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$id
#&gt; [1] 3
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$name
#&gt; [1] "BCR-ABL E255K"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$civic_actionability_score
#&gt; [1] 56
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$start
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$stop
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$variant_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$id
#&gt; [1] 98
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$name
#&gt; [1] "D842I"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$variant_types
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_bases
#&gt; [1] "GA"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$variant_bases
#&gt; [1] "AT"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$id
#&gt; [1] 99
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$name
#&gt; [1] "D842V"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$variant_types
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$civic_actionability_score
#&gt; [1] 100.5
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$id
#&gt; [1] 100
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$name
#&gt; [1] "D842Y"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$variant_types
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$stop
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$id
#&gt; [1] 101
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$name
#&gt; [1] "I843DEL"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$variant_types
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$civic_actionability_score
#&gt; [1] 5
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$start
#&gt; [1] 55152095
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_bases
#&gt; [1] "ATC"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$id
#&gt; [1] 102
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$name
#&gt; [1] "DI842-843VM"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$variant_types
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_bases
#&gt; [1] "ACATC"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$variant_bases
#&gt; [1] "TCATG"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$id
#&gt; [1] 241
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$name
#&gt; [1] "BCR-ABL F317L"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$description
#&gt; [1] "BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$civic_actionability_score
#&gt; [1] 99.5
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$start
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$stop
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[6]]$variant_groups[[1]]$type
#&gt; [1] "variant_group"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]
#&gt; $content$variants[[7]]$id
#&gt; [1] 101
#&gt; 
#&gt; $content$variants[[7]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[7]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[7]]$name
#&gt; [1] "I843DEL"
#&gt; 
#&gt; $content$variants[[7]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[7]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_types
#&gt; $content$variants[[7]]$variant_types[[1]]
#&gt; $content$variants[[7]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$variants[[7]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$variants[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$variants[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$variants[[7]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$variants[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$civic_actionability_score
#&gt; [1] 5
#&gt; 
#&gt; $content$variants[[7]]$coordinates
#&gt; $content$variants[[7]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[7]]$coordinates$start
#&gt; [1] 55152095
#&gt; 
#&gt; $content$variants[[7]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[7]]$coordinates$reference_bases
#&gt; [1] "ATC"
#&gt; 
#&gt; $content$variants[[7]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups
#&gt; $content$variants[[7]]$variant_groups[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$id
#&gt; [1] 1
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$name
#&gt; [1] "Imatinib Resistance"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$description
#&gt; [1] "While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$id
#&gt; [1] 2
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$name
#&gt; [1] "BCR-ABL T315I"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$civic_actionability_score
#&gt; [1] 105
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$start
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$stop
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$id
#&gt; [1] 3
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$name
#&gt; [1] "BCR-ABL E255K"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$civic_actionability_score
#&gt; [1] 56
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$start
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$stop
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$variant_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$id
#&gt; [1] 98
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$name
#&gt; [1] "D842I"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$variant_types
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_bases
#&gt; [1] "GA"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$variant_bases
#&gt; [1] "AT"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$id
#&gt; [1] 99
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$name
#&gt; [1] "D842V"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$variant_types
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$civic_actionability_score
#&gt; [1] 100.5
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$id
#&gt; [1] 100
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$name
#&gt; [1] "D842Y"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$variant_types
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$stop
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$id
#&gt; [1] 101
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$name
#&gt; [1] "I843DEL"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$variant_types
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$civic_actionability_score
#&gt; [1] 5
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$start
#&gt; [1] 55152095
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_bases
#&gt; [1] "ATC"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$id
#&gt; [1] 102
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$name
#&gt; [1] "DI842-843VM"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$variant_types
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_bases
#&gt; [1] "ACATC"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$variant_bases
#&gt; [1] "TCATG"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$id
#&gt; [1] 241
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$name
#&gt; [1] "BCR-ABL F317L"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$description
#&gt; [1] "BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$civic_actionability_score
#&gt; [1] 99.5
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$start
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$stop
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[7]]$variant_groups[[1]]$type
#&gt; [1] "variant_group"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]
#&gt; $content$variants[[8]]$id
#&gt; [1] 99
#&gt; 
#&gt; $content$variants[[8]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[8]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[8]]$name
#&gt; [1] "D842V"
#&gt; 
#&gt; $content$variants[[8]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[8]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_types
#&gt; $content$variants[[8]]$variant_types[[1]]
#&gt; $content$variants[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$civic_actionability_score
#&gt; [1] 100.5
#&gt; 
#&gt; $content$variants[[8]]$coordinates
#&gt; $content$variants[[8]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[8]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[8]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[8]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[8]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups
#&gt; $content$variants[[8]]$variant_groups[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$id
#&gt; [1] 1
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$name
#&gt; [1] "Imatinib Resistance"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$description
#&gt; [1] "While imatinib has shown to be incredibly successful in treating philadelphia chromosome positive CML, patients that have shown primary or secondary resistance to the drug have been observed to harbor T315I and E255K ABL kinase domain mutations. These mutations, among others, have been observed both in primary refractory disease and acquired resistance. In gastrointestinal stromal tumors (GIST), PDGFRA 842 mutations have also been shown to confer resistance to imatinib. "
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$id
#&gt; [1] 2
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$name
#&gt; [1] "BCR-ABL T315I"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$civic_actionability_score
#&gt; [1] 105
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$start
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$stop
#&gt; [1] 133748283
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[1]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$id
#&gt; [1] 3
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$name
#&gt; [1] "BCR-ABL E255K"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$description
#&gt; [1] "While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$civic_actionability_score
#&gt; [1] 56
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$start
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$stop
#&gt; [1] 133738363
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$variant_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[2]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$id
#&gt; [1] 98
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$name
#&gt; [1] "D842I"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$variant_types
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_bases
#&gt; [1] "GA"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$variant_bases
#&gt; [1] "AT"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[3]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$id
#&gt; [1] 99
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$name
#&gt; [1] "D842V"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. Exogenous expression of the A842V mutation resulted in constitutive tyrosine phosphorylation of PDGFRA in the absence of ligand in 293T cells and cytokine-independent proliferation of the IL-3-dependent Ba/F3 cell line, both evidence that this is an activating mutation. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$variant_types
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$civic_actionability_score
#&gt; [1] 100.5
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$stop
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_bases
#&gt; [1] "A"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[4]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$id
#&gt; [1] 100
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$name
#&gt; [1] "D842Y"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$variant_types
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$start
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$stop
#&gt; [1] 55152092
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_bases
#&gt; [1] "G"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$variant_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[5]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$id
#&gt; [1] 101
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$name
#&gt; [1] "I843DEL"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$variant_types
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$id
#&gt; [1] 107
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$name
#&gt; [1] "inframe_deletion"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$display_name
#&gt; [1] "Inframe Deletion"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001822"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$description
#&gt; [1] "An inframe non synonymous variant that deletes bases from the coding sequence."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001822"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$civic_actionability_score
#&gt; [1] 5
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$start
#&gt; [1] 55152095
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_bases
#&gt; [1] "ATC"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$variant_bases
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[6]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$id
#&gt; [1] 102
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$entrez_name
#&gt; [1] "PDGFRA"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$entrez_id
#&gt; [1] 5156
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$name
#&gt; [1] "DI842-843VM"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$description
#&gt; [1] "PDGFRA D842 mutations are characterized broadly as imatinib resistance mutations. The DI842-843VM variant is the result of a double point mutation. This is most well characterized in gastrointestinal stromal tumors, but other cell lines containing these mutations have been shown to be resistant as well. In imatinib resistant cell lines, a number of other therapeutics have demonstrated efficacy. These include; crenolanib, sirolimus, and midostaurin (PKC412)."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$gene_id
#&gt; [1] 38
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$variant_types
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$civic_actionability_score
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome
#&gt; [1] "4"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$start
#&gt; [1] 55152093
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$stop
#&gt; [1] 55152097
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_bases
#&gt; [1] "ACATC"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$variant_bases
#&gt; [1] "TCATG"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript
#&gt; [1] "ENST00000257290.5"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[7]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$id
#&gt; [1] 241
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$entrez_name
#&gt; [1] "ABL1"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$entrez_id
#&gt; [1] 25
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$name
#&gt; [1] "BCR-ABL F317L"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$description
#&gt; [1] "BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$gene_id
#&gt; [1] 4
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$type
#&gt; [1] "variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$id
#&gt; [1] 47
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$name
#&gt; [1] "missense_variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$display_name
#&gt; [1] "Missense Variant"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$so_id
#&gt; [1] "SO:0001583"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$description
#&gt; [1] "A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[1]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001583"
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$id
#&gt; [1] 120
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$name
#&gt; [1] "transcript_fusion"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$display_name
#&gt; [1] "Transcript Fusion"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$so_id
#&gt; [1] "SO:0001886"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$description
#&gt; [1] "A feature fusion where the deletion brings together transcript regions."
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$variant_types[[2]]$url
#&gt; [1] "http://www.sequenceontology.org/browser/current_svn/term/SO:0001886"
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$civic_actionability_score
#&gt; [1] 99.5
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome
#&gt; [1] "9"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$start
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$stop
#&gt; [1] 133748288
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_bases
#&gt; [1] "T"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$variant_bases
#&gt; [1] "C"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript
#&gt; [1] "ENST00000318560.5"
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$chromosome2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$start2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$stop2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$representative_transcript2
#&gt; NULL
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$ensembl_version
#&gt; [1] 75
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$variants[[8]]$coordinates$reference_build
#&gt; [1] "GRCh37"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$variants[[8]]$variant_groups[[1]]$type
#&gt; [1] "variant_group"
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$type
#&gt; [1] "variant_group"
#&gt; 
#&gt; $content$lifecycle_actions
#&gt; $content$lifecycle_actions$last_modified
#&gt; $content$lifecycle_actions$last_modified$timestamp
#&gt; [1] "2015-10-21T20:35:02.784Z"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user
#&gt; $content$lifecycle_actions$last_modified$user$id
#&gt; [1] 41
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$name
#&gt; [1] "Nick Spies"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$last_seen_at
#&gt; [1] "2019-01-20T03:51:36.969Z"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$username
#&gt; [1] "NickSpies"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$role
#&gt; [1] "admin"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$avatar_url
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$avatars
#&gt; $content$lifecycle_actions$last_modified$user$avatars$x128
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=128"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$avatars$x64
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=64"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$avatars$x32
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$avatars$x14
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=14"
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$area_of_expertise
#&gt; [1] "Research Scientist"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$orcid
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$display_name
#&gt; [1] "NickSpies"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$created_at
#&gt; [1] "2015-06-12T18:13:16.508Z"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$url
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$twitter_handle
#&gt; [1] "@NickSpies13"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$facebook_profile
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$linkedin_profile
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$bio
#&gt; [1] "Nick Spies is a staff analyst at the McDonnell Genome Institute and an MD student at Washington University School of Medicine. He has made substantial contributions to the development of genome analysis tools and resources at the Genome Institute including the Drug-Gene Interaction Database. He is a founding member of the CIViC team, helping to define the CIViC data model, and a leading content curator and a feature development consultant."
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$country
#&gt; $content$lifecycle_actions$last_modified$user$country$id
#&gt; [1] 214
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$country$iso
#&gt; [1] "US"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$country$name
#&gt; [1] "United States"
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$featured_expert
#&gt; [1] TRUE
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$accepted_license
#&gt; NULL
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$signup_complete
#&gt; NULL
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$affiliation
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization
#&gt; $content$lifecycle_actions$last_modified$user$organization$id
#&gt; [1] 1
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization$name
#&gt; [1] "The McDonnell Genome Institute"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization$url
#&gt; [1] "http://genome.wustl.edu/"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization$description
#&gt; [1] "The McDonnell Genome Institute (MGI) is a world leader in the fast-paced, constantly changing field of genomics. A truly unique institution, we are pushing the limits of academic research by creating, testing, and implementing new approaches to the study of biology with the goal of understanding human health and disease, as well as evolution and the biology of other organisms."
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization$profile_image
#&gt; $content$lifecycle_actions$last_modified$user$organization$profile_image$x256
#&gt; [1] "/system/organizations/profile_images/000/000/001/x256/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization$profile_image$x128
#&gt; [1] "/system/organizations/profile_images/000/000/001/x128/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization$profile_image$x64
#&gt; [1] "/system/organizations/profile_images/000/000/001/x64/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization$profile_image$x32
#&gt; [1] "/system/organizations/profile_images/000/000/001/x32/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization$profile_image$x14
#&gt; [1] "/system/organizations/profile_images/000/000/001/x14/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$last_modified$user$organization$parent
#&gt; named list()
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$created
#&gt; $content$lifecycle_actions$created$timestamp
#&gt; [1] "2015-06-21T16:49:38.943Z"
#&gt; 
#&gt; $content$lifecycle_actions$created$user
#&gt; $content$lifecycle_actions$created$user$id
#&gt; [1] 41
#&gt; 
#&gt; $content$lifecycle_actions$created$user$name
#&gt; [1] "Nick Spies"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$last_seen_at
#&gt; [1] "2019-01-20T03:51:36.969Z"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$username
#&gt; [1] "NickSpies"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$role
#&gt; [1] "admin"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$avatar_url
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$avatars
#&gt; $content$lifecycle_actions$created$user$avatars$x128
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=128"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$avatars$x64
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=64"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$avatars$x32
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$avatars$x14
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=14"
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$created$user$area_of_expertise
#&gt; [1] "Research Scientist"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$orcid
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$created$user$display_name
#&gt; [1] "NickSpies"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$created_at
#&gt; [1] "2015-06-12T18:13:16.508Z"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$url
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$created$user$twitter_handle
#&gt; [1] "@NickSpies13"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$facebook_profile
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$created$user$linkedin_profile
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$created$user$bio
#&gt; [1] "Nick Spies is a staff analyst at the McDonnell Genome Institute and an MD student at Washington University School of Medicine. He has made substantial contributions to the development of genome analysis tools and resources at the Genome Institute including the Drug-Gene Interaction Database. He is a founding member of the CIViC team, helping to define the CIViC data model, and a leading content curator and a feature development consultant."
#&gt; 
#&gt; $content$lifecycle_actions$created$user$country
#&gt; $content$lifecycle_actions$created$user$country$id
#&gt; [1] 214
#&gt; 
#&gt; $content$lifecycle_actions$created$user$country$iso
#&gt; [1] "US"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$country$name
#&gt; [1] "United States"
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$created$user$featured_expert
#&gt; [1] TRUE
#&gt; 
#&gt; $content$lifecycle_actions$created$user$accepted_license
#&gt; NULL
#&gt; 
#&gt; $content$lifecycle_actions$created$user$signup_complete
#&gt; NULL
#&gt; 
#&gt; $content$lifecycle_actions$created$user$affiliation
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization
#&gt; $content$lifecycle_actions$created$user$organization$id
#&gt; [1] 1
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization$name
#&gt; [1] "The McDonnell Genome Institute"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization$url
#&gt; [1] "http://genome.wustl.edu/"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization$description
#&gt; [1] "The McDonnell Genome Institute (MGI) is a world leader in the fast-paced, constantly changing field of genomics. A truly unique institution, we are pushing the limits of academic research by creating, testing, and implementing new approaches to the study of biology with the goal of understanding human health and disease, as well as evolution and the biology of other organisms."
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization$profile_image
#&gt; $content$lifecycle_actions$created$user$organization$profile_image$x256
#&gt; [1] "/system/organizations/profile_images/000/000/001/x256/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization$profile_image$x128
#&gt; [1] "/system/organizations/profile_images/000/000/001/x128/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization$profile_image$x64
#&gt; [1] "/system/organizations/profile_images/000/000/001/x64/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization$profile_image$x32
#&gt; [1] "/system/organizations/profile_images/000/000/001/x32/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization$profile_image$x14
#&gt; [1] "/system/organizations/profile_images/000/000/001/x14/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$created$user$organization$parent
#&gt; named list()
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on
#&gt; $content$lifecycle_actions$last_commented_on$timestamp
#&gt; [1] "2015-10-21T20:35:21.498Z"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user
#&gt; $content$lifecycle_actions$last_commented_on$user$id
#&gt; [1] 41
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$name
#&gt; [1] "Nick Spies"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$last_seen_at
#&gt; [1] "2019-01-20T03:51:36.969Z"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$username
#&gt; [1] "NickSpies"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$role
#&gt; [1] "admin"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$avatar_url
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$avatars
#&gt; $content$lifecycle_actions$last_commented_on$user$avatars$x128
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=128"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$avatars$x64
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=64"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$avatars$x32
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=32"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$avatars$x14
#&gt; [1] "https://secure.gravatar.com/avatar/3376aeb8439c5ab3e5d72fa2eeed39e5.png?d=identicon&amp;r=pg&amp;s=14"
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$area_of_expertise
#&gt; [1] "Research Scientist"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$orcid
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$display_name
#&gt; [1] "NickSpies"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$created_at
#&gt; [1] "2015-06-12T18:13:16.508Z"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$url
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$twitter_handle
#&gt; [1] "@NickSpies13"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$facebook_profile
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$linkedin_profile
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$bio
#&gt; [1] "Nick Spies is a staff analyst at the McDonnell Genome Institute and an MD student at Washington University School of Medicine. He has made substantial contributions to the development of genome analysis tools and resources at the Genome Institute including the Drug-Gene Interaction Database. He is a founding member of the CIViC team, helping to define the CIViC data model, and a leading content curator and a feature development consultant."
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$country
#&gt; $content$lifecycle_actions$last_commented_on$user$country$id
#&gt; [1] 214
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$country$iso
#&gt; [1] "US"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$country$name
#&gt; [1] "United States"
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$featured_expert
#&gt; [1] TRUE
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$accepted_license
#&gt; NULL
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$signup_complete
#&gt; NULL
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$affiliation
#&gt; [1] ""
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$id
#&gt; [1] 1
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$name
#&gt; [1] "The McDonnell Genome Institute"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$url
#&gt; [1] "http://genome.wustl.edu/"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$description
#&gt; [1] "The McDonnell Genome Institute (MGI) is a world leader in the fast-paced, constantly changing field of genomics. A truly unique institution, we are pushing the limits of academic research by creating, testing, and implementing new approaches to the study of biology with the goal of understanding human health and disease, as well as evolution and the biology of other organisms."
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$profile_image
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$profile_image$x256
#&gt; [1] "/system/organizations/profile_images/000/000/001/x256/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$profile_image$x128
#&gt; [1] "/system/organizations/profile_images/000/000/001/x128/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$profile_image$x64
#&gt; [1] "/system/organizations/profile_images/000/000/001/x64/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$profile_image$x32
#&gt; [1] "/system/organizations/profile_images/000/000/001/x32/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$profile_image$x14
#&gt; [1] "/system/organizations/profile_images/000/000/001/x14/MGI_STANDARD4_logo_brown-example_v1b.png?1494525976"
#&gt; 
#&gt; 
#&gt; $content$lifecycle_actions$last_commented_on$user$organization$parent
#&gt; named list()
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; 
#&gt; $content$sources
#&gt; list()
#&gt; 
#&gt; 
#&gt; $url
#&gt; [1] "https://civicdb.org/api/variant_groups/1"
#&gt; 
#&gt; $response
#&gt; Response [https://civicdb.org/api/variant_groups/1]
#&gt;   Date: 2019-09-03 16:44
#&gt;   Status: 200
#&gt;   Content-Type: application/json; charset=utf-8
#&gt;   Size: 107 kB
#&gt; 
#&gt; 
#&gt; attr(,"class")
#&gt; [1] "civic_api"</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
    <h2>Contents</h2>
    <ul class="nav nav-pills nav-stacked">
      <li><a href="#arguments">Arguments</a></li>
      
      <li><a href="#value">Value</a></li>
      
      <li><a href="#examples">Examples</a></li>
    </ul>

  </div>
</div>

      <footer>
      <div class="copyright">
  <p>Developed by Andrew Duncan.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
   </div>

  

  </body>
</html>

